app_design.txt

```diff
@@ -0,0 +1,50 @@
+IMPORTANTE READ IT ALL PLX
```

Comment:
As an exercise rewrite that to Markdown and keep it professional, you can check examples in any big github repo in README.md file

Reply:
not done